contrib/signet/getcoins.py

```diff
@@ -49,4 +49,16 @@
 except:
     print('Unexpected error when contacting faucet:', sys.exc_info()[0])
     exit()
-print(res.text)
+
+# Display the output as per the returned status code
+if res:
+    # When the return code in between 200 and 400 i.e. successful
+    print(res.text)
+elif res.status_code == 400:
+    print('You can use the global faucet only once per 24hrs. Please try again later.')
+elif res.status_code == 404:
+    print('The specified faucet URL does not exist. Please check for any server issues/typo.')
+elif res.status_code == 504:
+    print('The specified signet address is not valid. Please check it and try again.')
```

Comment:
Not sure about printing `address not valid` for a status code that should be `504 Gateway Timeout`. There might be different reasons for a 504.

Reply:
Makes sense :)

Reply:
Actually, when 504 happens, it returns a huge chunk of raw HTML which might look weird if we print we as output.